src/Nri/Ui/Modal/V5.elm

```diff
@@ -114,49 +115,77 @@ update config msg model =
     Modal.update config msg model
 
 
+{-| -}
+close : Msg
+close =
+    Modal.close
+
+
+{-| Pass the id of the element that focus should return to when the modal closes.
+-}
+open : String -> Msg
+open =
+    Modal.open
+
+
 {-| -}
 type alias FocusableElementAttrs msg =
-    { onlyFocusableElement : List (Root.Attribute msg)
-    , firstFocusableElement : List (Root.Attribute msg)
-    , lastFocusableElement : List (Root.Attribute msg)
+    { onlyFocusableElement : List (Attribute msg)
+    , firstFocusableElement : List (Attribute msg)
+    , lastFocusableElement : List (Attribute msg)
     }
 
 
 {-| -}
 info :
-    { title : { visibleTitle : Bool, title : String }
+    { visibleTitle : Bool
+    , title : String
```

Comment:
👍

Reply:
Just to confirm that I understand:  this is a Bool * String instead of a Maybe String because we always use the title in the acciessibility labels even if it's not shown?

Reply:
Yes, exactly!